js/app.js

```diff
@@ -91,4 +140,12 @@ $(document).ready( function() {
 		var tags = $(this).find("input[name='tags']").val();
 		getUnanswered(tags);
 	});
+	$('.inspiration-getter').submit(function(e){
+		e.preventDefault(); //
+		//Zero old results
+		$('.results').html('');
+		//get value from user
+		var tag = $(this).find("input[name='answerers']").val();
```

Comment:
you should trim the input, that way you don't end up searching for `%20javascript%20%20 ` with all that spaces